File: parish360-core/src/main/java/org/parish360/core/dto/common/Place.java
package org.parish360.core.dto.common;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

@Data
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Place {
    private String location;
    private String city;
    private String state;
    private String country;
}

File: parish360_mobile/lib/core/common/entities/place.dart
class Place {
  final String? location;
  final String? city;
  final String? state;
  final String? country;

  Place({
    this.location,
    this.city,
    this.state,
    this.country,
  });

  factory Place.fromJson(Map<String, dynamic> json) {
    return Place(
      location: json['location'] as String?,
      city: json['city'] as String?,
      state: json['state'] as String?,
      country: json['country'] as String?,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'location': location,
      'city': city,
      'state': state,
      'country': country,
    };
  }
}